SConstruct

```diff
@@ -279,9 +279,13 @@ def scons():
     buildinfo = prereqs.get_build_info()
     buildinfo.gen_script('.build_vars.sh')
     buildinfo.save('.build_vars.json')
-    env.InstallAs("$PREFIX/TESTING/.build_vars.sh", ".build_vars.sh")
-    env.InstallAs("$PREFIX/TESTING/.build_vars.json", ".build_vars.json")
-    env.InstallAs("$PREFIX/lib/daos/VERSION", "VERSION")
+    env.InstallAs("$PREFIX/lib/daos/TESTING/.build_vars.sh", ".build_vars.sh")
```

Comment:
You don't need to use InstallAs unless you are changing the name of the file....so this could simply be 

env.Install("$PREFIX/lib/daos/TESTING", ".build_vars.sh")
same for other lines.

This type of code could benefit from a function, maybe in daos_build

daos_build.install(env, "lib64/daos", file)

where said function is
def install(env, subdir, files): #files can be string or an array
    """install files in standard location
    path = "$PREFIX/%s" % subdir
    env.Install(path, files)

Not really your fault but I don't like repetition.